wiki/Tournaments/CWC/2020/en.md

```diff
@@ -0,0 +1,190 @@
+   - This may change depending on how many teams are accepted into the competition at the end.
+2. All the teams from each group will face each other.
+3. Rankings of each group are determined by sorting the results of each team's performance in the following priority:
+   - Most matches won.
+   - Have higher `{(the number of beatmaps won) - (the number of beatmaps defeated)}`.
+   - Most beatmaps won.
+   - **Winner of the match played previously between the tied teams.**
+   - In the event of a triple tie:
+     - Have higher `∑{(total score difference) / (maximum score)}`.
+     - Winner of the rematch.
+4. The top 2 teams of each group will move on to the Knock-Out Stages.
+   - This may change with the actual Group Stage setup.
+5. Following stages are Double Elimination Stages. This means that the winner moves to the next stage and the losing team gets moved to the Loser bracket.
+6. Based on [this image](https://puu.sh/bUq5V/f1066103b0.png), the stages are split up into the following:
+
+| Stage | Match ID |
+| --: | :-- |
+| Round of 16 |  A, B, C, D, E, F, G, H |
+| Quarterfinals | I, J, K, L & R, S, T, U |
+| Semifinals | M, N & V, W, X, Y, Z, AA |
+| Finals | O & AB, AC, AD |
+| Grand Finals | AE, P, Q |
+
+**Winning conditions:**
+   - In Group Stage, you need to win 5 maps to win a match. (Best-of-9)
+   - In the Round of 16 and the Quarterfinals, you need to win 6 maps to win a match. (Best-of-11)
+   - In the **Semifinals, Finals and Grand Finals**, you need to win 7 maps to win the match. (Best-of-13)
+
+### Match Instructions
+
+1. A referee will create a multiplayer room 15 minutes in advance. Players must gather during this period.
+   - Room settings are osu!catch, Team-Vs., Win Condition: 'ScoreV2'. Room name must be "CWC 2020: (TeamRed) vs (TeamBlue)".
+   - The team mentioned first in the room name must be the red team, the team mentioned second in the room name must be the blue team.
+2. Each team is free to select up to one warm-up map. Using beatmaps with questionable content is prohibited. All maps must be osu!catch specific maps.
+3. Each captain has to ban **one beatmap** to be selected from the pool. These beatmaps are not allowed to be picked by any team in the entire match.
+4. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool.
+5. Each captain must use "!roll" once in #multiplayer.
+   - The winner of the !roll starts picking the first beatmap of the match.
+   - The loser of the !roll starts banning one beatmap, followed by the winner of the !roll to ban a beatmap.
+6. Captains may pick freely from any bracket.
+   - In case of a tie, the tiebreaker map must be played.
+7. Teams will have 2 minutes to pick a beatmap and 2 minutes to get ready.
+   - In case a team takes more than 2 minutes to pick a map, they will recieve a warning from the referee, if that happens again the referee will pick a random map available from the mappool.
```

Comment:
What happens in the event a team picks the map on time, but takes more than 2 minutes to ready up? Warning -> Forfeit point?